### ISSUE TITLE
‘Next Watering Day’ attribute  is incorrect (one day off). 

### ISSUE THREAD

**Issue by DAB-LABS** [sebr/bhyve-home-assistant#261]
I’ve been using this integration for a while now without issue. 

I’m noticing ‘next watering day’ attribute is one day later than the actual next watering day assigned by BHyve. 

I have reloaded the integration and rebooted Home Assistant and this persists. 
![IMG_9938](https://github.com/user-attachments/assets/bb3b532b-de50-4b82-bd93-e40fbc722fe6)
![IMG_9939](https://github.com/user-attachments/assets/4fb6666a-fc0d-4090-b68f-0c2cfc5c4e1a)

**Comment by sebr** [comment]
Please upload your device diagnostics

**Comment by DAB-LABS** [comment]
Sure! Where can I get that info?

**Comment by sebr** [comment]
Did you not create an issue using the bug template? It has all the info in it.


> Please attach the Home Assistant diagnostics for the B-hyve device. Sensitive information is redacted when compiling the diagnostic data.
> 
> * Home Assistant diagnostics: https://www.home-assistant.io/integrations/diagnostics/
> * Diagnostic data implementation: https://github.com/sebr/bhyve-home-assistant/blob/main/custom_components/bhyve/diagnostics.py
> * Example of how to generate: https://user-images.githubusercontent.com/81972/179869298-3d2a33ca-f847-4a25-8e77-20757e49f914.png

**Comment by DAB-LABS** [comment]
Thank you. Yes, I overlooked the template. Apologies. 

The device is an indoor BHyve WT24-001 with firmware 0053. Here's the diag:

[config_entry-bhyve-a2fbfccbc530987f5ee61e54a423e796.json](https://github.com/user-attachments/files/16983594/config_entry-bhyve-a2fbfccbc530987f5ee61e54a423e796.json)

**Comment by DAB-LABS** [comment]
The next watering date changed to Feb 2016 (in HA) for a few days while showing a different watering day in the BHyve app (next week). As of yesterday, the next watering dates now match (HA and Orbit App).  Odd…

**Comment by DAB-LABS** [comment]
The date has been proper over the last two weeks. I did have to reload the integration hourly in order to pickup on any changes in the next watering day as the integration seems to query less (daily?). 

I don't see a need to look into this as a bug. Maybe I observed the Orbit decision tree in real time...? 

Would be cool to add an API interval for the query to Orbit for people want realtime data... 

Thanks for checking this out!
